packages/svelte/src/lib/T.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+  import getTolgeeContext from "./getTolgeeContext";
+  import { onDestroy } from "svelte";
+
+  export let keyName: string;
+  export let parameters: Record<string, any> | undefined = undefined;
+  export let strategy: "ELEMENT_WRAP" | "TEXT_WRAP" | "NO_WRAP" =
+    "ELEMENT_WRAP";
+
+  if (!keyName) {
+    console.error("Missing keyName prop!");
+    // show the error in the DOM
+    keyName = "Tolgee: Missing key name prop!";
```

Comment:
Is this really a good idea? I already see how it will be visible in production app. Wouldn't it be better to just show nothing and print error to console? Also typescript should prevent this, no?